app/assets/stylesheets/govuk_publishing_components/components/_contents-list.scss

```diff
@@ -50,7 +50,7 @@
   padding-left: $contents-spacing;
   padding-right: $contents-spacing;
 
-  & span::before {
+  & .gem-c-contents-list__list-item-dash::before {
```

Comment:
With the introduction of this class, I think this can now be further simplified. This line can change to:

```Sass
.gem-c-contents-list__list-item-dash::before {
```

...and the new section you added on line 84 can be moved out of the 2nd nested level and done as a single child of the parent, i.e.

```Sass
.gem-c-contents-list--alternative-line-style {
  .gem-c-contents-list__list-item {
    padding: govuk-spacing(2) 0;
    margin: 0;
  }

  .gem-c-contents-list__list-item-dash::before {
    display: none;
  }
```

Reply:
Thanks. I think you can also remove the `&` from the start of this line now as well.

Reply:
Darn, I missed that during my rebase, fixed now. @andysellick